experimental/packages/exporter-trace-otlp-grpc/test/OTLPTraceExporter.test.ts

```diff
@@ -181,9 +179,16 @@ const testCollectorExporter = (params: TestParams) =>
           let spans;
           let resource;
           if (exportedData) {
-            spans = exportedData.instrumentationLibrarySpans[0].spans;
+            spans = exportedData.scopeSpans[0].spans;
             resource = exportedData.resource;
-            ensureExportedSpanIsCorrect(spans[0]);
+
+            assert.ok(
+              typeof spans !== 'undefined',
+              'spans do not exist'
+            );
+            if(spans) {
+              ensureExportedSpanIsCorrect(spans[0]);
```

Comment:
TypeScript should discriminate the type of spans to be non-undefined since we have an assertion above. Is this `if (spans)` needed? Seems like it is totally fine without the if-branch, like lines 242-246.

Reply:
You're right, that shouldn't be needed. Removed in [02c4c55](https://github.com/open-telemetry/opentelemetry-js/pull/2929/commits/02c4c5555cd7d5996769bd759ad067666d1a2b0b). Thanks! :slightly_smiling_face: